ee/app/livechat-enterprise/lib/messageTypes.js

```diff
@@ -0,0 +1,18 @@
+import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
+
+import { MessageTypes } from '../../../../app/ui-utils';
+
+MessageTypes.registerType({
+	id: 'livechat_priority_history',
+	system: true,
+	message: 'New_chat_priority',
+	data(message) {
+		if (!message.priorityData) {
+			return;
+		}
+		return {
+			user: message.priorityData.definedBy.username,
```

Comment:
I would destructure the `message.priorityData` properties to make the code more readable.